R/unit.R

```diff
@@ -380,9 +381,9 @@ pSummary <- function(..., op) {
     x <- unclass(x)
     x[i] <- value
     if (simpleResult) {
-        `attributes<-`(x, attr)
+        attributes(x) <- attr
     } else {
-        `class<-`(x, "unit")
+        class(x) <- "unit"
     }
```

Comment:
What is the reason for this change? doesn't it prevent the object from being returned?

Reply:
Oops.  Thanks for spotting that.  I missed returning 'x' at the end of the function.  Nice demonstration of why I should have been providing you with pull requests all along :)  
This change is now hopefully part of the pull request (?)

Reply:
Are you happy to merge this so that I can just generate a single patch from the 'new-unit' branch to commit to r-devel ?

Reply:
I'll merge it into #1